include/openthread/message.h

```diff
@@ -105,6 +105,21 @@ typedef struct otThreadLinkInfo
     uint8_t mRadioType; ///< Radio link type.
 } otThreadLinkInfo;
 
+/**
+ * Allocates and returns a (generic) message buffer.
+ *
+ * For IPv6 messages use of `otIp6NewMessage()` is recommended over this function.
+ *
+ * @note If @p aSettings is 'NULL', the link layer security is enabled and the message priority is set to
```

Comment:
```suggestion
 * @note If @p aSettings is NULL, the link layer security is enabled and the message priority is set to
```

Since we don't use quotes elsewhere

Reply:
Thanks @Vyrastas. Made all the updates in the new pushed commit.